webcompat/templates/__init__.py

```diff
@@ -31,8 +31,10 @@ def bust_cache(file_path):
     if app.config['LOCALHOST']:
         return file_path
     if file_path.startswith('/'):
-        file_path = file_path[1:]
-    absolute_path = os.path.join(STATIC_PATH, file_path)
+        trimmed_file_path = file_path[1:]
```

Comment:
not an expert, but maybe a bit shorter version:
```python
    trimmed_file_path = file_path
    if file_path.startswith('/'):
        trimmed_file_path = file_path[1:]

    absolute_path = os.path.join(STATIC_PATH, trimmed_file_path)

```

Reply:
oh right, nicer. :)